@uportal/waffle-menu/README.md

```diff
@@ -0,0 +1,252 @@
+      flex-wrap: wrap;
+      padding: .5rem 1rem;
+    }
+
+    .buttons {
+      margin-left: 90%;
+    }
+  </style>
+  <script>
+    // JSON to be used to pass to <waffle-menu> through the data attribute
+    let json = [
+      {
+        link: 'https://blackboard.cpp.edu/?index=1',
+        image: 'https://www.cpp.edu/img/icons/sq-BlackBoard.png',
+        label: 'Blackboard',
+        type: "box"
+      },
+      {
+        link: 'https://outlook.office365.com/cpp.edu?index=1',
+        image: 'https://www.cpp.edu/img/icons/sq-Email.png',
+        label: 'Email',
+        type: "box"
+      },
+      {
+        link: 'https://idp.cpp.edu/idp/profile/cas/login?service=https://cmsweb.cpp.edu/psp/HPOMPRDF/EMPLOYEE/HRMS/c/SA_LEARNER_SERVICES.SSS_MY_PLANNER.GBL?gsmobile=0&amp;Page=SSS_MY_PLANNER&amp;Action=A',
+        image: 'https://www.cpp.edu/img/icons/sq-MyPlanner.png',
+        label: 'My Planner',
+        type: "box"
+      },
+      {
+        link: 'https://idp.cpp.edu/idp/profile/cas/login?service=https://cmsweb.cpp.edu/psp/HPOMPRDF/EMPLOYEE/HRMS/s/WEBLIB_PTPP_SC.HOMEPAGE.FieldFormula.IScript_AppHP?pt_fname=POM_BD_SS&amp;FolderPath=PORTAL_ROOT_OBJECT.POM_BD_SS&amp;IsFolder=true?cmd=start&amp;languageCd=ENG&amp;userid=PS&amp;pwd=z&amp;calstateEduPersonEmplID',
+        image: 'https://www.cpp.edu/img/icons/sq-BroncoDirect.png',
+        label: 'BroncoDirect',
+        type: "box"
+      },
+      {
+        link: 'https://cmsweb.cpp.edu/psp/HPOMPRD/EMPLOYEE/HRMS/c/SA_LEARNER_SERVICES.SSS_STUDENT_CENTER.GBL?pslnkid=POM_BD_STDNT_SC&amp;PORTALPARAM_PTCNAV=POM_BD_STDNT_SC&amp;EOPP.SCNode=HRMS&amp;EOPP.SCPortal=EMPLOYEE&amp;EOPP.SCName=POM_BD_SS&amp;EOPP.SCLabel=Student%20Self%20Services&amp;EOPP.SCFName=POM_BD_STDNT&amp;EOPP.SCSecondary=true&amp;EOPP.SCPTfname=POM_BD_STDNT&amp;FolderPath=PORTAL_ROOT_OBJECT.POM_BD_SS.POM_BD_STDNT.POM_BD_STDNT_SC&amp;IsFolder=false',
+        image: 'https://www.cpp.edu/img/icons/sq-Lynda.png',
+        label: 'Lynda.com',
+        type: "box"
+      },
+      {
+        link: 'https://blackboard.cpp.edu/?index=1',
+        image: 'https://www.cpp.edu/img/icons/sq-StudentCenter.png',
+        label: 'Student Center',
+        type: "box"
+      },
+      {
+        link: 'http://www.cpp.edu/~test-group/portal/online-services.shtml?index=1',
+        label: 'View all online services',
+        type: "footer"
+      },
+      {
+        link: 'https://google.com',
+        label: 'Google Search',
+        type: "footer"
+      },
+    ];
+
+    // since we're passing data through an HTML attribute, 
+    // gotta make the object a string.
+    let jsonString = JSON.stringify(json);
+  </script>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="buttons">
+        <waffle-menu buttoncolor="#ffffff" />
+      </div>
+    </nav>
+  </header>
+  <script>
+    // uncomment for loading JSON through the 'data' attribute
+    document.querySelector('waffle-menu').setAttribute('data', jsonString);
```

Comment:
It would be nice to have a wrapper to directly call the uPortal layout.
Having the design asset is good, having the design asset and a high level API is nicer. :slightly_smiling_face: